src/circuit.mjs

```diff
@@ -310,6 +310,19 @@ export class HeadlessCircuit {
         ret.subcircuits = subcircuits;
         return ret;
     }
+    getSvgExportUrl(circuitSvgDOMElement) {
+        // Remove useless in export jointJS tooltips
+        circuitSvgDOMElement
+            .querySelectorAll('foreignObject.tooltip')
+            .forEach(obj => obj.remove());
```

Comment:
If I understand the intention right, this modifies the SVG element of a JointJS paper. If this is used on an existing paper in an UI, doesn't this "damage" it?

Reply:
After changing the parameter to `joint.dia.paper` it is no longer needed to remove the tooltips, so this is not relevant anymore.